### PR TITLE
Updated (And Hopefully Working) Vision 

### PR DESCRIPTION
**Pull request by first5188**

### CODE REVIEW

File: src/main/java/frc/robot/hardware/intake/IntakeIOSparkFlex.java [hunk 1]
```diff
@@ -51,6 +51,13 @@ public void setTargetPositionAsDegrees(double degrees) {
         _pivotMotor.getPIDController().setReference(Units.degreesToRotations(degrees), ControlType.kPosition);
     }
 
+    public void setIsRollerRolling(boolean isRollerRolling) {
```
Comment (rhit-edgingza) [inline src/main/java/frc/robot/hardware/intake/IntakeIOSparkFlex.java:54]: We don't need this method

File: src/main/java/frc/robot/subsystems/climber/Climber.java [hunk 2]
```diff
@@ -15,19 +16,6 @@ public boolean getCanMove() {
     return _canMove;
   }
 
-  public void setClimberSpeed(double speed) {
```
Comment (rhit-edgingza) [inline src/main/java/frc/robot/subsystems/climber/Climber.java:18]: Please fix the code so we don't have any of this commented out

File: src/main/java/frc/robot/subsystems/intake/commands/CmdIntakeRollersSpit.java [hunk 3]
```diff
@@ -16,23 +16,16 @@ public CmdIntakeRollersSpit(Intake intakeSubsystem) {
 
     @Override
     public void initialize() {
-        _intakeSubsytem.setRollerSpeedSpit();
```
Comment (rhit-edgingza) [inline src/main/java/frc/robot/subsystems/intake/commands/CmdIntakeRollersSpit.java:19]: This command is incorrect now, please put back in the original code that runs the rollers

File: src/main/java/frc/robot/subsystems/intake/commands/CmdIntakeSetPosition.java [hunk 4]
```diff
@@ -19,10 +19,9 @@ public CmdIntakeSetPosition(Intake intakeSubsystem, IntakePosition intakePositio
         addRequirements(intakeSubsystem);
     }
 
-    // Called when the command is initially scheduled.
+    // // Called when the command is initially scheduled.
     @Override
     public void initialize() {
-        _intakeSubsystem.setIntakePosition(_intakePosition);
```
Comment (rhit-edgingza) [inline src/main/java/frc/robot/subsystems/intake/commands/CmdIntakeSetPosition.java:25]: You're missing the call to the subsystem again

File: src/main/java/frc/robot/subsystems/intake/commands/CmdIntakeStartRollersAcquire.java [hunk 5]
```diff
@@ -14,7 +14,6 @@ public CmdIntakeStartRollersAcquire(Intake intakeSubsystem) {
 
     @Override
     public void initialize() {
-        _intakeSubsystem.setRollerMotorSpeedAcquire();
```
Comment (rhit-edgingza) [inline src/main/java/frc/robot/subsystems/intake/commands/CmdIntakeStartRollersAcquire.java:17]: Missing subsystem call

File: src/main/java/frc/robot/subsystems/intake/commands/CmdIntakeStopRollers.java [hunk 6]
```diff
@@ -15,7 +15,6 @@ public CmdIntakeStopRollers(Intake intakeSubsystem) {
 
     @Override
     public void initialize() {
-        _intakeSubsystem.stopRollerMotor();
```
Comment (rhit-edgingza) [inline src/main/java/frc/robot/subsystems/intake/commands/CmdIntakeStopRollers.java:18]: Missing subsystem call

**Comment by first5188** [comment]
Okay I put in the Climber.java fix

**Review by rhit-edgingza — CHANGES_REQUESTED** [review]

**Comment by mitchgres** [comment]
Whenever the fixed tests are put into main we can fix the commented out test but since we aren't using sim-classes anymore the tests being commented out is needed for it to build.

**Review by rhit-edgingza — APPROVED** [review]

**Pull request merged**